service/handler.go

```diff
@@ -112,7 +113,7 @@ func (svc *Service) handleMessage(message kafka.Message) (string, error) {
 		}
 
 		for _, request := range lineDimensions {
-			if err := request.Post(http.DefaultClient); err != nil {
+			if err := request.Post(svc.HTTPClient.HTTPClient); err != nil {
```

Comment:
this needs to be `Post(ctx, svc.HTTPClient)` - the rchttp client alongside the context